manifests/init.pp

```diff
@@ -318,6 +321,7 @@
   Variant[Integer[2, 3], Boolean]      $use_schleuder         = false,         # postfix_use_schleuder
   Boolean                              $use_sympa             = false,         # postfix_use_sympa
   Hash                                 $virtuals              = {},
+  Hash                                 $canonicals            = {},
```

Comment:
Any chance you can make the Hash datatype a bit stricter?

Reply:
I'd rather not make the `Any` more tied to the particulars of the `postfix::canonical` - that Any gets validated when given to `postfix::canonical`.